### ISSUE TITLE
Crash while sharing

### ISSUE THREAD

**Issue by ecamejop91** [nextcloud/android#7970]
### Steps to reproduce
1. Select a folder
2.try to share the folder with another NC user 
3. 

### Expected behaviour
- Tell us what should happen
The folders should be shared 
### Actual behaviour
Crash
- Tell us what happens
Just crashed
### Can you reproduce this problem on https://try.nextcloud.com? 
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill 
some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by Armag3ddon** [comment]
Can confirm this. Here's my crash log:

************ CAUSE OF ERROR ************

java.lang.RuntimeException: Accessing result data after operation failed!
	at com.owncloud.android.lib.common.operations.RemoteOperationResult.getData(RemoteOperationResult.java:486)
	at com.owncloud.android.utils.ErrorMessageAdapter.getMessageForCreateShareOperations(ErrorMessageAdapter.java:254)
	at com.owncloud.android.utils.ErrorMessageAdapter.getMessageForResultAndOperation(ErrorMessageAdapter.java:143)
	at com.owncloud.android.utils.ErrorMessageAdapter.getErrorCauseMessage(ErrorMessageAdapter.java:79)
	at com.owncloud.android.ui.activity.FileActivity.onRemoteOperationFinish(FileActivity.java:380)
	at com.owncloud.android.ui.activity.FileDisplayActivity.onRemoteOperationFinish(FileDisplayActivity.java:1704)
	at com.owncloud.android.services.OperationsService.lambda$dispatchResultToOperationListeners$0(OperationsService.java:734)
	at com.owncloud.android.services.-$$Lambda$OperationsService$KsB1LbjHjlEBSp0h7XO24czs-OQ.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7660)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30150090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: google
Device: sargo
Model: Pixel 3a
Id: RQ1A.210105.002
Product: sargo

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 6985033

**Comment by AndyScherzinger** [comment]
Thanks for the error log, Looping in @tobiasKaminsky

**Comment by ecamejop91** [comment]
Hello, thank you for your answer.
Basically the whole log is the same, just some numbers at the end of the
lines are slightly different.
But here  will send you a screenshot
of what I got.
Thank you for your time.

**Comment by AndyScherzinger** [comment]
@ecamejop91 thanks for your answer unfortunately Github doesn't add screenshots within mail replies but just the text 😢

**Comment by ecamejop91** [comment]
Here is the error log:





2:52 O n MO O EO

Nextcloud crashed

************ CAUSE OF ERROR ************

java.lang.RuntimeException: Accessing result data after operation failed! at

com.owncloud.android.lib.common.operations. Remote Operation
Result.getData(Remote Operat ion Result.java:486)

at com.owncloud.android.utils.Error MessageAdap
ter.getMessageForCreateShareOperations(Error MessageAdapter.java:254) at

com.owncloud.android.utils.ErrorMessageAdap
ter.getMessageForResultAndOperation(ErrorMe ssageAdapter.java:143)

at com.owncloud.android.utils.Error MessageAdap
ter.getErrorCauseMessage(ErrorMessageAdapt

er.java:79)

at

com.owncloud.android.ui.activity.FileActivity.on Remote
OperationFinish(FileActivity.java:380)

at

com.owncloud.android.ui.activity.FileDisplay Act ivity.onRemote Operation
Finish(FileDisplayActivi

ty.java:1704)

at com.owncloud.android.services. Operations Serv
ice.lambda$dispatchResultToOperationListener

s$0(Operations Service.java:734) at com.owncloud.android.services.-$

$Lambda$Operations Service$KSB1LbjHjIEBSP Oh7X024czs-0Q.run(Unknown
Source:6) at

android.os.Handler.handleCallback(Handler.jav a:883) at

android.os.Handler.dispatchMessage(Handler.j

ava:100)

at android.os.Looper.loop(Looper.java:237)

at

android.app.ActivityThread.main(ActivityThread

.java:8167)

at java.lang.reflect.Method.invoke(Native

Method)

at

com.android.internal.os.Runtimelnit$MethodAn
dArgsCaller.run(RuntimeInit.java:496) at

com.android.internal.os.ZygoteInit.main(Zygote Init.java:1100)

************ APP INFORMATION ************

ID: com.nextcloud.client

Version: 30150090 Build flavor: gplay

************ DEVICE INFORMATION ************

Brand: samsung

Device: crownqltesq Model: SM-N960U

Id: QP1A.190711.020

Product: crownqltesq

SDK: 29

Release: 10

Incremental: N960U SQS 6 FT 1

**Comment by tobiasKaminsky** [comment]
Duplicate of https://github.com/nextcloud/android/issues/7987
Fix is in: #7992
